### PR TITLE
Add Hungarian translation strings

### PR DESCRIPTION
**Pull request by nagyrobi**

### CODE REVIEW

**Comment by nagyrobi** [comment]
Can you please add a `label` for "(master)" string too?

**Comment by kalkih** [comment]
Sure, I've added `label.master` as a new translation string, see #380.
Feel free to update your PR with this translation.

File: src/translations.js [hunk 1]
```diff
@@ -37,6 +37,25 @@ const translations = {
...
+  hu: {
+    placeholder: {
+      tts: 'Szövegfelolvasás',
+    },
+    label: {
+      leave: 'Kilépés',
+      ungroup: 'Összes kilép',
+      group_all: 'Összes belép',
+      send: 'Küldés',
+      master: 'forrás',
```
Comment (kalkih) [inline src/translations.js:49]: Please capitalize this, that way we can easily reuse the same string in the future, without the need for extra styling rules.

Comment (nagyrobi) [reply src/translations.js:49]: I was about to notice that it shouldn't be capitalised in English either, but I accept your approach.

**Pull request merged**